src/elements/DestructiveDialog/DestructiveDialog.js

```diff
@@ -0,0 +1,45 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button';
+
+const DestructiveDialog = ({
+	title,
+	content,
+	leftButtonClickHandler,
```

Comment:
Let's rename these "leftButton..."/"rightButton..." to be "primaryAction..."/"secondaryAction..."

The layout could change, mobile might do a stacked style, etc so a more generic but still specific naming would be better.